portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,17 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Chooses a random cat photo to display on page
  */
```

Comment:
nit: This can be a single line comment with /** and */ on the same line. Also super nit, generally function comments are sentences it's good to keep the period